### DISCUSSION TITLE
CHAPTER09andahalf\webapp --ScrapeWithPandas.ipynb --'records== gazpacho_records'

### DISCUSSION THREAD

**Discussion by lpf1573** [headfirstpython/third#6] [General]
when I compare ,'records==gazpacho_recordss',output is 'False',which not the same with the book. I don't know why .I confirm the data is no different between records and gazpach_records

**Comment by barryp-work** [comment]
Sorry for the delay in responding - I have been away and offline for about 10 days.

The `gazpacho_records` data is taken from the `records.json` file which (if downloaded from the book's website) may be out of date as the wikipedia page is constantly updated with new swimming world records (for example, after a major swimming event such as the recent Paris Olympics).  Be sure to regenerate a fresh (and up-to-date) `records.json` file using the `update_records.py` program from the end of Chapter 9, then copy the updated `records.json` file over the stale file in your most-recent `webapp` folder.  When you do, your comparison will report `True` (and all should be well with the world once more).   

--Paul.
